features/ActiveRecord.md

```diff
@@ -0,0 +1,181 @@
+# Active Record
```

Comment:
I'd prefer it if you reverted this change, if you want to add to the documentation in another PR or in the original file thats fine, but this is really hard to review what if anything you've changed